### DISCUSSION TITLE
app for armeabi-v7a not woks on Android 6.0

### DISCUSSION THREAD

**Discussion by areful** [rustdesk/rustdesk#3811] [Q&A] [answered]
I found that apk for armeabi-v7a cannot work on Android 6.0, when app start running, it only shows a white screen.
 (But it works well on Android 12)

apk from: 
[rustdesk-1.2.0-armv7-linux-androideabi-release-signed.apk](https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.2.0-armv7-linux-androideabi-release-signed.apk)

**Comment by rustdesk** [comment, marked as answer]
wont fix Android 6.0. Frankly, we have no armv7 phone, we build it with Github CI, so please do not report issues about v7, we have no way to test and fix.

**Reply by areful** [reply]
Thanks :)

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/issues/743#issuecomment-1489736796
